schema.sql

```diff
@@ -4,8 +4,9 @@ CREATE TABLE animals (
     id INT GENERATED ALWAYS AS IDENTITY,
     name VARCHAR(255) NOT NULL,
     date_of_birth DATE NOT NULL,
-    escape_attempts INT NOT NULL CHECK (escape_attempts >= 0),
+    escape_attempts INT NOT NULL CHECK,
     neutered BOOLEAN NOT NULL,
-    weight_kg FLOAT(2) NOT NULL CHECK (weight_kg >= 0),
+    weight_kg FLOAT(2) NOT NULL CHECK,
+    species VARCHAR(255) NOT NULL,
```

Comment:
- [ ] in order to change an existing table you should use the `ALTER_TABLE` query instead of changing the table itself.

Reply:
That's true 
![image](https://user-images.githubusercontent.com/105387310/206173413-1d033325-87cc-4bb8-aa82-71e414512915.png)
Delete line 10 will keep the table like before.
Then proceed to change the table using ALTER TABLE:
![image](https://user-images.githubusercontent.com/105387310/206176723-48a11431-94a5-4564-8f91-c28062c15705.png)

![image](https://user-images.githubusercontent.com/105387310/206174250-f7a6ed14-3f45-4fbe-8ed4-9eebf459ae12.png)
I will add this to the Pull request